### ISSUE TITLE
Add animations and .gif support

### ISSUE THREAD

**Issue by Logianer** [sebinside/StreamAwesome#126]
Add support for simple/basic animations such as:
- bounce
- shake
- spin left/right
- pulse fade

...and automagically export these icons with their animations as a .gif-image.

**Comment by sebinside** [comment]
Yes, I would argue that this has very low priority as animated icons are more of a gimmick and an anit-pattern as they steal your attention instead of speeding up your workflow.
